commit 09834d6cfa193439f6b35364b4f1c6134643198a
Author: Yaakov Selkowitz <yselkowi@redhat.com>
Date:   Sun Jun 23 12:09:14 2024 -0400

    Fix compilation with GCC 14
    
    C++20 has changed the syntax for nested template constructors:
    
    ```
    In file included from ../../src/vma/proto/mem_buf_desc.h:39,
                     from ../../src/vma/util/utils.h:47,
                     from vlogger.cpp:46:
    ../../src/vma/util/vma_list.h:210:31: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
      210 |         vma_list_t<T, offset> (const vma_list_t<T, offset>& other) {
          |                               ^
    ../../src/vma/util/vma_list.h:210:31: note: remove the '< >'
    ```
    
    Signed-off-by: Yaakov Selkowitz <yselkowi@redhat.com>

diff --git src/vma/util/vma_list.h src/vma/util/vma_list.h
index cd3e5f9eef38..550fe709ea05 100644
--- src/vma/util/vma_list.h
+++ src/vma/util/vma_list.h
@@ -207,7 +207,7 @@ public:
 		}
 	}
 
-	vma_list_t<T, offset> (const vma_list_t<T, offset>& other) {
+	vma_list_t(const vma_list_t<T, offset>& other) {
 		if (!other.empty())
 			vlist_logwarn("Copy constructor is not supported for non-empty list! other.size=%zu", other.m_size);
 		init_list();
